abis/ForwarderFactory.json

```diff
@@ -0,0 +1,63 @@
+[
+    {
+        "inputs": [
+            {
+                "internalType": "address",
+                "name": "_implementationAddress",
+                "type": "address"
+            }
+        ],
+        "stateMutability": "nonpayable",
+        "type": "constructor"
+    },
+    {
+        "anonymous": false,
+        "inputs": [
+            {
+                "indexed": false,
+                "internalType": "address",
+                "name": "newForwarderAddress",
+                "type": "address"
+            },
+            {
+                "indexed": false,
+                "internalType": "address",
+                "name": "parentAddress",
+                "type": "address"
+            }
+        ],
+        "name": "ForwarderCreated",
+        "type": "event"
+    },
+    {
+        "inputs": [],
+        "name": "implementationAddress",
+        "outputs": [
+            {
+                "internalType": "address",
+                "name": "",
+                "type": "address"
+            }
+        ],
+        "stateMutability": "view",
+        "type": "function"
+    },
+    {
+        "inputs": [
+            {
+                "internalType": "address",
+                "name": "parent",
+                "type": "address"
+            },
+            {
+                "internalType": "bytes32",
+                "name": "salt",
+                "type": "bytes32"
+            }
+        ],
+        "name": "createForwarder",
+        "outputs": [],
+        "stateMutability": "nonpayable",
```

Comment:
I guess this "stateMutability": "nonpayable" is the same as "payable": false used in other files?

Reply:
Hm I'm not sure exactly. These are auto-generated files